Core/Inc/datalogging/odometer_save.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * odometer_save.h
+ *
+ *  Created on: May 8, 2022
+ *      Author: dmroh
+ */
+
+#ifndef INC_DATALOGGING_ODOMETER_SAVE_H_
+#define INC_DATALOGGING_ODOMETER_SAVE_H_
+
+#define SAVE_DELAY 30000
+
+#define ODOMETER_DATA_FLASH_ADDR 0x080E0000
+
+#define WORDS_READ_TWO 2
+
+#define MAXIMUM_SAVE_TIMES 20
+
+#define MINIMUM_SAVE_DISTANCE 100
+
+enum { TOTAL_DISTANCE, FLASH_WEAR };
```

Comment:
```suggestion
enum { TOTAL_DISTANCE = 0, FLASH_WEAR };
```

isso é uma boa prática, já que vc usa o enum como indexador do seu array mais abaixo